doc/nrf/releases_and_maturity/releases/release-notes-2.9.0.rst

```diff
@@ -0,0 +1,642 @@
+
+  * Updated to handle generic CME errors from all ``AT%CMNG`` commands.
+
+nRF RPC libraries
+-----------------
+
+* Added the :ref:`nrf_rpc_dev_info` library for obtaining information about a device connected through the :ref:`nrfxlib:nrf_rpc`.
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+* :ref:`nrf_desktop_config_channel_script` Python script:
+
+  * Added support for pure ED25519 signature (used by nRF54L based devices that enable MCUboot hardware cryptography).
+    This requires using ``imgtool`` supporting pure ED25519 signature that can be installed from ``sdk-mcuboot`` repository.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``a4eda30f5b0cfd0cf15512be9dcd559239dbfc91``, with some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
+* Added an option that allows to select the number of KMU key slots (also known as generations) to use when verifying an image.
+  See MCUboot's Kconfig option :kconfig:option:`CONFIG_BOOT_SIGNATURE_KMU_SLOTS`.
+
+Zephyr
+======
+
+.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each nRF Connect SDK release.
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``beb733919d8d64a778a11bd5e7d5cbe5ae27b8ee``, with some |NCS| specific additions.
+
+For the list of upstream Zephyr commits (not including cherry-picked commits) incorporated into nRF Connect SDK since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
+
+.. code-block:: none
+
+   git log --oneline beb733919d ^ea02b93eea
+
+For the list of |NCS| specific commits, including commits cherry-picked from upstream, run:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^beb733919d
+
+The current |NCS| main branch is based on revision ``beb733919d`` of Zephyr.
+
+.. note::
+   For possible breaking changes and changes between the latest Zephyr release and the current Zephyr version, refer to the :ref:`Zephyr release notes <zephyr_release_notes>`.
+
+Documentation
+=============
+
+* Added:
+
+  * The :ref:`matter_samples_config` page that documents Kconfig options and snippets shared by Matter samples and applications.
+  * A page about :ref:`add_new_driver`.
+  * A page for the :ref:`sdp_gpio` application.
+
+* Updated the structure and contents of :ref:`config_gpio_pin` with more detailed information.
```

Comment:
This is rendered as a Kconfig option in the PR preview. The solution to this would be to add a new label on the "Driving a GPIO pin directly" page that doesn't start with "config_", and provide that as ref here.
Not sure, if this is just for the developer docs (PR preview) or same is seen in the Zoomin docs as well.

Reply:
Trying :ref:`Driving a GPIO pin directly <config_gpio_pin>`

Reply:
I think the label in the source document need to be modified into something like gpio_pin_config and that label should be used here.

Reply:
Looks like that. Luckily, the label has not been used anywhere yet. Will change the label.